src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+@import '/src/styles/blocks/stars';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: $text-secondary-color;
+}
+
+body {
+  margin: 0;
+}
+
+.cards__card {
+  margin-right: 48px;
+  margin-bottom: 46px;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
consider moving this property to universal * selector

Reply:
Not fixed
![2023-12-20_15h24_51](https://github.com/mate-academy/layout_catalog/assets/61904995/7d3949cc-60a3-4620-8e71-62dceff3434f)